Sources/SuitEtecsaSdk/entity/nauta/auth/LoginRequest.swift

```diff
@@ -2,8 +2,8 @@ import Foundation
 import Alamofire
 
 struct LoginRequest: Codable, Sendable {
-    let username, password, accountType, idRequest: String
-    let captchatext: String
+    public let username, password, accountType, idRequest: String
+    public let captchatext: String
```

Comment:
```suggestion
    let username, password, accountType, idRequest: String
    let captchatext: String
```

public no es necesario